packages/meditrak-server/src/routes/importEntities/importEntities.js

```diff
@@ -15,9 +15,13 @@ import { assertCanImportEntities } from './assertCanImportEntities';
  */
 export async function importEntities(req, res) {
   try {
-    const { models } = req;
+    const { models, query } = req;
+
     let entitiesByCountryName;
-    const pushToDhis = req?.query?.pushToDhis ? req?.query?.pushToDhis === 'true' : true;
+    const pushToDhis = query?.pushToDhis ? query?.pushToDhis === 'true' : true;
+    const automaticallyFetchGeojson = query?.automaticallyFetchGeojson
+      ? query?.automaticallyFetchGeojson === 'true'
+      : true;
```

Comment:
This is a) a bit funky, and b) could be DRYer given they both do the same thing. Might be good to have generic handling of boolean options coming from the front end?